### ISSUE TITLE
TF Node backend does not support normalized logits passed to multinomial

### ISSUE THREAD

**Issue by AndreMaz** [tensorflow/tfjs#2672]
#### TensorFlow.js version
V1.5.1

#### Describe the problem or feature request
Just stumbled on this error:
`TF Node backend does not support normalized logits passed to multinomial`

https://github.com/tensorflow/tfjs/blob/c2d556415db8d9fb611a69a77a6288678ab0f86d/tfjs-node/src/nodejs_kernel_backend.ts#L1561-L1568

I know that as a workaround I can unnormalize the inputs, but what's the actual limitation? Why the node backend doesn't support this?

**Comment by kangyizhang** [comment]
This is because TensorFlow C library multinomial op only support unnormalized logits: https://www.tensorflow.org/api_docs/cc/class/tensorflow/ops/multinomial

FYI tfjs-node library is using TensorFlow C library under the hood.
